libs/kano-profile/Makefile

```diff
@@ -53,4 +67,11 @@ clean-debug:
 	@rm -f $(DEBUG_OUTDIR)/$(OBJS)
 	@rm -f $(DEBUG_OUTDIR)/$(SONAME)
 
-clean: clean-release clean-debug
+link: all  # for Mac OS
+	mkdir -p /usr/local/include/kano/profile
+	ln -fs `pwd`/includes/* /usr/local/include/kano/profile
+	ln -fs `pwd`/release/$(SONAME) /usr/local/lib
```

Comment:
Can we be a little less aggressive with the linky links? Maybe do a check for existence first and only do it if it doesn't exist.